src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Razor/RazorSourceUpdater.cs

```diff
@@ -38,7 +45,8 @@ public RazorSourceUpdater(IEnumerable<DiagnosticAnalyzer> analyzers, IEnumerable
             _analyzers = analyzers?.OrderBy(a => a.SupportedDiagnostics.First().Id) ?? throw new ArgumentNullException(nameof(analyzers));
             _codeFixProviders = codeFixProviders?.OrderBy(c => c.FixableDiagnosticIds.First()) ?? throw new ArgumentNullException(nameof(codeFixProviders));
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
-            _diagnostics = Enumerable.Empty<Diagnostic>();
+            _chunker = new CharacterChunker();
+            _differ = new Differ();
```

Comment:
This is an implementation of IDiffer that could be passed in via DI to simplify testing.

Reply:
Good point. I'll get it from there.